src/tutorials/custom_properties.md

```diff
@@ -0,0 +1,240 @@
+```
+
+This will change the value of the **Custom Property** to be newValue and will be replicated to the client.
+
+!!! note
+    The script will error if the new value is either not the same data type as the **Custom Property** or if the **Custom Property** is not networked.
+
+More information can be found in the [CoreObject API Reference Functions](/api/coreobject/#functions) and [CoreObject API Reference Examples](/api/coreobject/#setnetworkedcustomproperty).
+
+### Custom Property Changed Event
+
+When a **Custom Property** is changed, an event is fired on the object.
+
+To listen to this event of a CoreObject, type this in the script:
+
+```lua
+function OnCustomPropertyChanged(coreObject, customPropertyName)
+    local newValue = coreObject:GetCustomProperty(customPropertyName)
+
+    print(string.format("New value of %s for %s is now %s", customPropertyName, coreObject.name, newValue))
+end
+
+CorObject.networkedPropertyChangedEvent:Connect(OnCustomPropertyChanged)
+```
+
+More information can be found in the [CoreObject API Reference Events](/api/coreobject/#events) and [CoreObject API Reference Examples](/api/coreobject/#networkedpropertychangedevent).
+
+## Simple Data Types
+
+For more information on simple data types such as string, number, and boolean, check out our [Data Types Reference](/tutorials/scripting_intro/#data-types).
+
+The simple data types that are supported as custom properties are:
+
+- Bool
+- Color
+- Float
+- Int
+- Rotation
+- String
+- Vector2
+- Vector3
+- Vector4
+
+### Bool
+
+A **Boolean** is only ever `true` or `false`. This can be best compared to on/off, yes/no, etc.
+
+In custom properties, they are expressed as a check box that is either checked (`true`) or unchecked (`false`).
+
+**Booleans** are most often used in if-statements that allow you to write code that is only executed if certain conditions are met, such as the boolean being `true`.
+
+### Color
+
+A **Color** is a Core class that contains values for `red`, `green`, `blue`, and `alpha` (transparency).
+
+**Colors** are helpful constants that can assist in UI programming, spawning assets of different colors, etc.
+
+More information can be found in the [Color API](/api/color).
+
+### Float
+
+A **Float** is a number that can have decimals, such as `0.1`, `1.2`, `139.8`, etc.
+
+### Int
+
+An **Int** is a number that can not have decimals and must be whole, such as `0`, `1`, `139`, etc.
+
+### Rotation
+
+A **Rotation** is a Core class that contains values for `x`, `y`, and `z` on rotation axis.
+
+More information can be found in the [Rotation API](/api/rotation/).
+
+### String
+
+A **String** is a collection of any value you want, such as numbers, letters, punctuation, and emojis. They are represented in double quotes (" ") in code so that the computer does not mistake them for code. However, this is not necessary for custom properties as **String** custom properties are represented as a text box that can be typed in.
```

Comment:
```suggestion
A **String** is a collection of any value you want, such as numbers, letters, punctuation, and emojis. They are represented in double quotes (`" "`) in code so that the computer does not mistake them for code. However, this is not necessary for custom properties as **String** custom properties are represented as a text box that can be typed in.
```